Tests/TuistKitAcceptanceTests/ImplicitImportsLintCommandAcceptanceTests.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import TuistAcceptanceTesting
+
+final class ImplicitImportsLintCommandAcceptanceTests: TuistAcceptanceTestCase {
```

Comment:
```suggestion
final class LintImplicitImportsAcceptanceTests: TuistAcceptanceTestCase {
```

Now that I think about it, I'd also rename the `ImplicitImportsLintCommand` to `LintImplicitImportsCommand` as that follows how it's used in the CLI.